index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>SUSE Openstack Cloud</title>
```

Comment:
not sure it really matters, but this may impact localization of the title

Reply:
This wasn't localized in the html file before this change, this file is based on the one generated by the webpack plugin we were using.